workspaces/github-pull-requests-board/plugins/github-pull-requests-board/src/utils/functions.ts

```diff
@@ -157,3 +158,18 @@ export const shouldDisplayCard = (
   // when "team" filter is toggled off, only shows PR on team repos
   return repositories.includes(fullRepoName);
 };
+
+export const decorateCommitStatus = (status: Status[]) => {
```

Comment:
Thanks for the quick review!

1. I forgot those were an option
2. The other parts of the plugin use emoji for status decorators, so it would be consistent with the rest of the plugin
3. I wasn't able to get a test-environment running to make sure it works, so sticking with a string seemed least likely to cause problems

But I'm happy to give it a go!

Reply:
I'm not a React dev, but it looks like to use the status components I'd have to pull the evaluation logic out of `functions.ts` because they are React components, which the pure typescript can't handle?

Reply:
@awanlin I'm not seeing an easy way to switch to use the status components. Would you consider approving without them or do you see that as a hard requirement?

Reply:
Sorry this took me so long to follow up on. I really wanted to pull this down and see how it looked in action (I went down an Uffizzi rabbit hole sadly). I think your initial point about the plugin using emoji is a good call out and should be the path forward. 👍 

On a purely academic note you should have been able to change the file extension to `.tsx` imported React and be able to work from there.